### DISCUSSION TITLE
Feature request: search UX/UI enhancements

### DISCUSSION THREAD

**Discussion by llemeurfr** [edrlab/thorium-reader#1761] [Ideas]
A heavy user from a large publishing house noticed the following: 

- You have to type your search term and then press the enter key. Most UIs now allow you to press one of the arrow buttons in the Search dialogue box to begin searching in that direction. Thorium requires pressing enter, and starting the search from the beginning of the book.
- The dialogue box does say how many matches were found, but that number doesn’t increment/decrement as you click the arrows to view each match. In many browsers and ereaders there is a number (e.g. 34/237, 35/237 etc).
- Results are frequently invisible to the user because they fall at the top or bottom of the viewport and are covered by Thorium’s own UI. This is part of a wider issue where I sometimes see content cut off by the UI. Screenshot attached – note the green identification box is hidden by the search box. I regularly see content in Thorium at the bottom of the viewport appearing cut-off, but in this instance it’s the top.

All this will be open to discussion at the time we have a large review of the app design.

**Comment by danielweck** [comment]
This meta-issue isn't really actionable at the moment. The details need to be fleshed out ... moving to discussions.
